### ISSUE TITLE
NullReferenceException at MsgReader.Outlook.Storage.Message.SetEmailSenderAndRepresentingSender()

### ISSUE THREAD

**Issue by TonyValenti** [Sicos1977/MSGReader#284]
**Describe the bug**

When attempting to parse some email files, I get an NullReferenceException at
```
   at MsgReader.Outlook.Storage.Message.SetEmailSenderAndRepresentingSender()
   at MsgReader.Outlook.Storage.Message.LoadStorage(CFStorage storage)
   at MsgReader.Outlook.Storage..ctor(Stream storageStream, FileAccess fileAccess)
   at MsgReader.Outlook.Storage.Message..ctor(Stream storageStream, FileAccess fileAccess)
```

**To Reproduce**
Parse certain emails.  I'll email you an example MSG.

**Expected behavior**
The email parses as expected.

**Desktop (please complete the following information):**
Windows 11 x64

**Comment by TonyValenti** [comment]
@Sicos1977 - I emailed you an example MSG file that causes this error.

**Comment by ShantoLives** [comment]
Started getting this in the 4.2.2 update

**Comment by TonyValenti** [comment]
@Sicos1977 any chance you can fix this asap?  It is really causing issues for us.

**Comment by Sicos1977** [comment]
I don't have any problems opening that e-mail, no error nothing

![image](https://user-images.githubusercontent.com/6692947/142148430-2d906bd3-e257-494d-b317-ef42b6d1d892.png)

**Comment by Sicos1977** [comment]
Fixed in 4.2.3 that I just published on nuget
